hudi-common/src/test/java/org/apache/hudi/common/fs/inline/TestInLineFileSystemHFileInLining.java

```diff
@@ -56,11 +58,12 @@
  */
 public class TestInLineFileSystemHFileInLining {
 
+  private static final String LOCAL_FORMATTER = "%010d";
+  private static final String VALUE_PREFIX = "value";
+  private static final int MIN_BLOCK_SIZE = 1024;
```

Comment:
Can we add suffix to clarify what measure this is (bytes, kbytes, etc)

Reply:
Fixed.